package com.aspireapp.aspiretest.exception;

public class AspireException extends RuntimeException {

    public AspireException(String message, Throwable cause) {
        super(message, cause);
    }
}
